f2/src/__tests__/ConfirmCancelPage.test copy.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
lol this file needs a better name! "ConfirmCancelPage.test copy.js" indeed.

Reply:
🙄